commands/open.ts

```diff
@@ -1,24 +1,29 @@
-// @ts-nocheck
-import { EXIT_CODES } from '../lib/enums/exitCodes';
-const {
+import { Argv, ArgumentsCamelCase } from 'yargs';
+import {
   addAccountOptions,
   addConfigOptions,
   addUseEnvironmentOptions,
   addGlobalOptions,
-} = require('../lib/commonOpts');
-const { trackCommandUsage } = require('../lib/usageTracking');
-const { logSiteLinks, getSiteLinksAsArray, openLink } = require('../lib/links');
-const { promptUser } = require('../lib/prompts/promptUtils');
-const { i18n } = require('../lib/lang');
+} from '../lib/commonOpts';
+import { trackCommandUsage } from '../lib/usageTracking';
+import { logSiteLinks, getSiteLinksAsArray, openLink } from '../lib/links';
+import { promptUser } from '../lib/prompts/promptUtils';
+import { i18n } from '../lib/lang';
+import { EXIT_CODES } from '../lib/enums/exitCodes';
+import {
+  CommonArgs,
+  ConfigArgs,
+  EnvironmentArgs,
+  AccountArgs,
+} from '../types/Yargs';
 
 const i18nKey = 'commands.open';
 
 const separator = ' => ';
-const createListPrompt = async accountId =>
-  promptUser([
+async function createListPrompt(accountId: number) {
```

Comment:
Missing return type here!

Reply:
Done in [577341a](https://github.com/HubSpot/hubspot-cli/pull/1401/commits/577341aac466df1118386022e479b9809a1104d1)